packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -107,6 +107,16 @@ class EditorPage {
 		await typeString( this.driver, block, text, clear );
 	}
 
+	async typeKeyString( inputString ) {
```

Comment:
In comparison to the existing `typeString`, is this new method an attempt to more closely simulate typing or some other purposes?

Reply:
Using the other util `typeString` Aztec wasn't able to detect the changes and apply the formatting styles. If I'm not mistaken we've only had tests where we would apply formatting styles to a selected text (which would be handled by the RichText component) but starting to type with a formatting style enabled wasn't working with `typeString`